lib/charms/tls_certificates_interface/v1/tls_certificates.py

```diff
@@ -1146,7 +1155,7 @@ def _on_relation_changed(self, event: RelationChangedEvent) -> None:
         if not self._relation_data_is_valid(provider_relation_data):
             logger.warning(
                 f"Provider relation data did not pass JSON Schema validation: "
-                f"{event.relation.data[event.app]}"
+                f"{event.relation.data[relation.app]}"
```

Comment:
Is there any difference?

Reply:
In the data no, but for type checking, `event.app` is an `Optional`. Also, we use `relation.app` further up in this method, so this just standardize the method a bit and fix type checking with `ops>=1.5.3`.

Reply:
Got it. Didn't think of types. Thanks!